sql/src/test/java/io/crate/integrationtests/SysNodeResiliencyIntegrationTest.java

```diff
@@ -77,4 +76,16 @@ public void testTimingOutNode() throws Exception {
             waitNoPendingTasksOnAll();
         }
     }
+
+    /**
+     * Test cluster name check, when path.data does not contain cluster
+     * name as a folder.
+     *
+     * TODO-ES6: Remove this when ES >= 6.0
+     */
+    @Test
+    public void testClusterNameInPathCheckDoesNotFailWithDefaultConfig() throws Exception {
```

Comment:
i'm going to add an extra assertion that cluster name folder really doesn't exist.

How would you like to verify that the default config is okay? And should this be part of this test case?